### DISCUSSION TITLE
DB2 for IBM i Add Schema to Schema Browser Error

### DISCUSSION THREAD

**Discussion by gcraig88keys** [codefori/vscode-ibmi#2218] [IBM i] [answered]
I'm getting an error - **Cannot read properties of undefined (reading 'includes')** when attempting to open the Schema "Manage Schema Browser List" hamburger menu on the DB2 for IBM i Schema Browser.   

I've tried to uninstall the entire developer pack and reinstall with no success.  I'm trying to avoid uninstalling VSCode.  Any ideas?

![image](https://github.com/user-attachments/assets/78cc2a3b-79b3-4bb6-8546-8ddc1629922f)

**Comment by worksofliam** [comment]
Don't have an answer for you but I can take a look first thing Monday morning.

**Reply by worksofliam** [reply]
@gcraig88keys Any chance you can share a video of this happening? I've been trying to recreate it with no luck.

**Reply by gcraig88keys** [reply, marked as answer]
This is so strange.  When I went to record a video of it, it worked just fine and the schema list on the server appeared.  But it wasn't working this morning.   If it happens again, I'll post a video.
